### ISSUE TITLE
Unable to install in VSCode 1.44.0

### ISSUE THREAD

**Issue by cwaldren** [rust-lang/rls-vscode#760]
I have a brand new install of VSCode 1.44.0. 

I attempted to install the Rust (rls) extension, but it fails with an error:

![Screen Shot 2020-04-08 at 11 03 37 PM](https://user-images.githubusercontent.com/5474173/78862914-48929f00-79ed-11ea-88b9-ba51b700e742.png)

![Screen Shot 2020-04-08 at 11 03 02 PM](https://user-images.githubusercontent.com/5474173/78862902-44668180-79ed-11ea-9953-587422526f97.png)

```
Error: the task 'Rust: Installing `rust-analysis`' neither specifies a command nor a dependsOn property. The task will be ignored. Its definition is:
{
    "type": "shell",
    "id": "shell,rustup component add rust-analysis --toolchain stable-x86_64-apple-darwin,",
    "problemMatcher": [],
    "label": "Rust: Installing `rust-analysis`"
}
```

```
Version: 1.44.0
Commit: 2aae1f26c72891c399f860409176fe435a154b13
Date: 2020-04-08T08:23:56.137Z
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Darwin x64 19.0.0
```

```
macOS Catalina (10.15.1)
```

**Comment by cdbfoster** [comment]
I encountered this on Linux too.  I downgraded to use VSCode 1.43, installed the extension successfully and then upgraded back to 1.44 without issue.

**Comment by ejconlon** [comment]
FWIW I had the same issue and manually ran `rustup component add rls rust-analysis rust-src`. When I opened VSCode again the extension seemed happy to use what I had installed.

**Comment by Relaxe111** [comment]
Hello i have similar issue too @ejconlon   helped me to solve the issue.Thnks

**Comment by wilfredinni** [comment]
> FWIW I had the same issue and manually ran `rustup component add rls rust-analysis rust-src`. When I opened VSCode again the extension seemed happy to use what I had installed.

can confirm it fixes the problem on Manjaro and vscodium too

**Comment by mfiumara** [comment]
I have the same problem on Manjaro Linux, had to manually install the rust-src and rls components.
This happens when one of the three components is already installed in the system, I do not have enough experience with VScode extensions to propose a PR but the problem lies somewhere in the `installRlsComponents` function.

**Comment by faisalr** [comment]
Fixed with  `rustup component add rls rust-src`

**Comment by faisalr** [comment]
it needs `rust-src` to go to definition

**Comment by Scratchcat1** [comment]
I had the same issue and setting `"rust-client.rlsPath": "rls"` in VScode's `settings.json` resolved the issue.

**Comment by Xanewok** [comment]
Fixed in https://github.com/rust-lang/rls-vscode/commit/800a04afd2629146ccc18013ab36f978705d20cf, released a 0.7.2 with the fix. It seems that this has regressed in the newest VSCode.
